content/blog/posts/tji-travels-under-the-pink-dome.md

```diff
@@ -0,0 +1,28 @@
+---
+title: TJI Travels Under the Pink Dome
+subtitle: ""
+date: 2021-05-10
+authors:
+  - Eva Ruth Moravec
+---
+Every other year, the pink granite building that sits in the heart of Austin buzzes with activity for six months as lawmakers gather to debate, consider, toss out and approve legislation that governs life in Texas. 
+
+This is one of said years, and said lawmakers are currently tumbleweeding towards the finish line at the end of May, when the clock runs out for bills to make it to Gov. Greg Abbott's desk for one last vote. For the first time, I testified *for* two bills – both proposals that would enhance transparency and accountability in criminal justice. While neither is likely to become law this session, I'm grateful for the opportunity and learned a new thing or two about this sausage-making process of ours that I thought I'd share. 
```

Comment:
I'm curious to know how you got to participate in the session from the beginning.

Reply:
another great point!